src/common/isolatable_hardwares.cpp

```diff
@@ -1046,7 +1041,7 @@ std::optional<sdbusplus::message::object_path> IsolatableHWs::getInventoryPath(
                  this](const auto& path) {
                 return isolatedHwDetails->second._invPathFuncLookUp(
                     this->_bus, path, uniqIsolateHwKey);
-            });
+                });
```

Comment:
Adjust white space issue

Reply:
It got formatted by itself when I ran clang-format.

Reply:
Then run clang in a separate commit. Also why clang would change something which you didn't change?